PSReadLine/Movement.vi.multiline.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+
+namespace Microsoft.PowerShell
+{
+    public partial class PSConsoleReadLine
+    {
+        /// <summary>
+        /// Moves the cursor to the beginning of the first logical line
+        /// of a multi-line buffer.
+        /// </summary>
+        /// <param name="key" />
+        /// <param name="arg" />
+        public void MoveToFirstLogicalLine(ConsoleKeyInfo? key = null, object arg = null)

```

Comment:
Hmm, I hadn't considered the public functions using `LogicalLine`, just the private function.

This function name is exposed, e.g. you'd see it when listing key bindings or in the documentation. I'm thinking `Logical` isn't necessary here.

Reply:
Good catch. I've reverted the function names as instructed.